engine/api/src/main/java/io/deephaven/engine/table/ColumnDefinition.java

```diff
@@ -504,6 +504,28 @@ public void describeDifferences(@NotNull List<String> differences, @NotNull fina
         }
     }
 
+    /**
+     * Checks if {@link #getDataType() dataType} can be cast to {@code destDataType}. If not, this throws a
+     * {@link ClassCastException}.
+     *
+     * @param destDataType the destination data type
+     */
+    public final void checkCastTo(Class<?> destDataType) {
+        TypeHelper.checkCastTo("[" + name + "]", dataType, destDataType);
```

Comment:
This context being passed does not mesh well with the exception message as currently composed.